site/gatsby-site/src/templates/cite.js

```diff
@@ -230,8 +230,16 @@ function CitePage(props) {
       addToast({
         message: (
           <Trans i18n={i18n}>
-            Please <Link to={localizePath({ path: '/login', language: locale })}>log in</Link> to
-            subscribe
+            Please{' '}
+            <Link
+              to={localizePath({
+                path: `/login?redirectTo=${window.location.pathname + window.location.search}`,
+                language: locale,
+              })}
+            >
+              log in
+            </Link>{' '}
+            to subscribe
```

Comment:
Please fix the translations. They are broken after this change with `{' '}`

Spanish version:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/6564809/224134948-510cf04d-d3a7-460a-8e2b-eb3e9647f956.png">